src/main/java/com/robothand/highqualitybot/command/NowPlayingCommand.java

```diff
@@ -29,12 +30,19 @@ public String getShortDesc() {
     @Override
     public void onCommand(MessageReceivedEvent event, String[] args) {
         GuildMusicPlayer musicPlayer;
+        AudioTrack track;
         Guild guild = event.getGuild();
         MessageChannel channel = event.getChannel();
+        StringBuilder message = new StringBuilder();
```

Comment:
Might be good to call this messageBuilder or something just for good self-documenting codr